sso-kit-client/src/SingleSignOnContext.ts

```diff
@@ -167,44 +176,35 @@ export class SingleSignOnContext {
   };
 
   /**
-   * Logouts from the application and redirects the user to the authentication provider's login page.
-   */
-  loginAgain = async () => {
-    await logout();
-    window.location.href = this.loginUrl!;
-  };
-
-  /**
-   * Logouts from the application and clears the user's authentication information.
+   * Logs out from the application and the authentication provider.
```

Comment:
This sentence is only valid if the `redirectUrl` is not provided and it defaults to `logoutUrl`, or the `redirectUrl` equals to `logoutUrl`. In other cases there is no logout on the provider side.

Could something similar be better?
 ```suggestion
Logs out from the application and redirects the user to the redirect URL if provided.
If not provided then the redirect URL defaults to {@link #logoutUrl} which logs out the user from the authentication provider too.
If the redirect URL is set to `undefined` then only the application logout happens without any redirect.
```

Reply:
That is true, since the provided logout URL should already have the redirection embedded in it.

That means it's not necessary to pass it as a parameter, as it is configured setting the `vaadin.sso.logout-redirect-route` in `application.properties`.

I've removed the parameter.